hypothesis-python/src/hypothesis/strategies/_internal/misc.py

```diff
@@ -43,9 +43,13 @@ def value(self):
         return self.elements[0]
 
     def __repr__(self):
+        suffix = "".join(
+            f".{name}({get_pretty_function_description(f)})"
+            for name, f in self._transformations
+        )
```

Comment:
We could potentially find a way to unify this with the mostly-identical code up in `SampledFromStrategy.__repr__`.

Though if you just want to get this simple fix in as-is then I'm open to that as well.

Reply:
I tried that, but it turned out much simpler this way.